.github/workflows/release-please.yml

```diff
@@ -12,7 +12,7 @@ name: release-please
 
 jobs:
   release-please:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
I think we can keep `ubuntu-latest` here, given that we are just running release-please, and not any of REANA's code